app/src/main/java/com/seoulfitu/android/ui/sports_service_list/SportsServiceListActivity.kt

```diff
@@ -0,0 +1,59 @@
+package com.seoulfitu.android.ui.sports_service_list
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import androidx.activity.viewModels
+import androidx.core.widget.doOnTextChanged
+import com.seoulfitu.android.databinding.ActivitySportsServiceListBinding
+import com.seoulfitu.android.ui.sports_service_detail.SportsServiceDetailActivity
+import com.seoulfitu.android.ui.sports_service_list.viewmodel.SportsServiceListViewModel
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class SportsServiceListActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivitySportsServiceListBinding
+    private val viewModel: SportsServiceListViewModel by viewModels()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        initBinding()
+        observeUiState()
+        setUpView()
+        viewModel.getServices()
+    }
+
+    private fun initBinding() {
+        binding = ActivitySportsServiceListBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+    }
+
+    private fun observeUiState() {
+        viewModel.uiState.observe(this) {
+            when (it.isSuccess) {
+                true -> {
+                    binding.rvSportsServiceList.apply {
+                        adapter = SportsServiceAdapter(it.result) { sportsService ->
+                            SportsServiceDetailActivity.start(context, sportsService)
+                        }
+                    }
```

Comment:
apply를 사용하신 이유가 있을까요? adapter 하나만을 지정하고 있어 스코프 함수를 사용할 필요가 있나 싶습니다. 중괄호를 한 번 더 써서 오히려 더 복잡해보이는 것 같아요..!
```suggestion
                    binding.rvSportsServiceList.adapter = SportsServiceAdapter(it.result) { sportsService ->
                            SportsServiceDetailActivity.start(context, sportsService)
                        }
```

Reply:
이게 더 깔끔해 보이네용 ㅎㅎ 수정하겠습니다🤭🤭